ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -294,60 +308,102 @@ gcp_file_open(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name
 static int
 gcp_remove(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, uint32_t flags)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
-    WT_UNUSED(name);
-    WT_UNUSED(flags);
-
-    return 0;
+    std::cerr << "gcp_remove: object: " << name << ": file removal is not supported." << std::endl;
+    return ENOTSUP;
 }
 
 static int
 gcp_rename(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *from, const char *to,
   uint32_t flags)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
-    WT_UNUSED(from);
-    WT_UNUSED(to);
-    WT_UNUSED(flags);
+    std::cerr << "gcp_rename: object: " << from << ": file renaming is not supported." << std::endl;
+    return ENOTSUP;
+}
+
+static int
+gcp_object_size(WT_FILE_SYSTEM *file_system, [[maybe_unused]] WT_SESSION *session, const char *name,
+  wt_off_t *sizep)
```

Comment:
`size_t` works better here since we're only dealing with positive values. `size_t` is an unsigned int type whereas `wt_off_t` is a signed type.

Reply:
To my understanding we need to use wt_off_t here since it is mapping to a WiredTiger function.